app/include_upgrade_mainnet.go

```diff
@@ -0,0 +1,20 @@
+//go:build mainnet
```

Comment:
I remember Go allows to build with multiple tags e.g., `-tags "mainnet,testnet"`. Wondering what will happen in this case :thinking:

Reply:
good question, in both cases I am replacing the upgrades var from app.go with a new upgrade 
So, the question is which one ? :sweat_smile: 

One other point is that in the makefile or it includes mainnet or testnet to the build tags 
https://github.com/babylonlabs-io/babylon/blob/aea173fe43423b42918d8f827976856c69179239/Makefile#L82-L87